pages/sitemap.xml.ts

```diff
@@ -1,13 +1,19 @@
 import { GetServerSideProps } from 'next';
 
 function generateSiteMap() {
-    const rootUrl = process.env.VERCEL_URL
-        ? process.env.VERCEL_URL
-        : 'http://localhost:3000';
+    let baseUrl = 'http://localhost:3000';
+    if (process.env.VERCEL_ENV) {
+        baseUrl = {
+            production: 'https://theblackhart.co.uk',
```

Comment:
This is a placeholder URL - this should be updated to `https://torchbox.com/careers/`